task.go

```diff
@@ -72,13 +72,11 @@ func (e *Executor) Run(ctx context.Context, calls ...taskfile.Call) error {
 	for _, call := range calls {
 		task, err := e.GetTask(call)
 		if err != nil {
-			e.ListTasks(FilterOutInternal(), FilterOutNoDesc())
-			return err
+			return fmt.Errorf("task: executor.Run(): executor.GetTask(): %v", err)
```

Comment:
No need to wrap the error here. The extra context is not useful to the user and it's not making the test break.

```suggestion
			return err
```

What is making some tests break is the printing of tasks on `e.ListTasks`. With your change, it's not being print automatically anymore when the user tries to run a task that does not exist. That needs to be added back. Maybe in another place? I don't know, need to figure it out where.

Reply:
I tried putting this
```
_, _ = e.ListTasks(ListOptions{ListOnlyTasksWithDescriptions: true})
```
in place of restoring
```
e.ListTasks(FilterOutInternal(), FilterOutNoDesc())
```
but now two more tests are breaking, expecting nothing where they are now getting
```
task: No tasks with description available. Try --list-all to list all tasks\n
```

I think that the old `ListTasks(...)` method didn't print anything if it just filtered out all the tasks while the new one picked up that "try --list-all" method from somewhere else.

probably I tried reducing duplication and tripped over two places in the code that were almost the same but not quite.

what do you think?